rocketpy/monte_carlo/DispersionModel.py

```diff
@@ -0,0 +1,171 @@
+__author__ = "Mateus Stano Junqueira, Guilherme Fernandes Alves"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+
+from pydantic import BaseModel, Extra, root_validator, validator
+from ..tools import get_distribution
+
+
+class DispersionModel(BaseModel):
+    """Base class for all dispersion models. This class is used to validate
+    the input parameters of the dispersion models, based on the pydantic library.
+    """
+
+    class Config:
+        """Configures pydantic to allow arbitrary types and extra fields."""
+
+        arbitrary_types_allowed = True
+        extra = Extra.allow
+
+    def __repr__(self):
+        field_values = "\n\t"
+        field_values += "\t".join(
+            [
+                f"{field_name}={getattr(self, field_name)!r}\n"
+                for field_name in self.__fields__
+            ]
+        )
+        return f"{self.__class__.__name__}({field_values})\n"
+
+    # TODO: find a way to validate if distribution_function string is the name
```

Comment:
Is this really necessary?

Reply:
Well... it is already done hehe

It was done in https://github.com/RocketPy-Team/RocketPy/pull/336/commits/0d5fa7072d0fd15871992bee9366ebedb2a5a828